src/integrations/PinterestTag/browser.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable class-methods-use-this */
+import sha256 from 'sha256';
```

Comment:
We have already used sha256 from `crypto-js`, can't we use that one here?

Reply:
used sha256 from `crypto-js`